components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AccessTokenDAOImpl.java

```diff
@@ -239,28 +239,37 @@ private void insertAccessToken(String accessToken, String consumerKey, AccessTok
             } else {
                 insertTokenPrepStmt.setString(18, NONE);
             }
+
+            String authorizedOrganization = accessTokenDO.getAuthzUser().getAccessingOrganization();
+            if (StringUtils.isEmpty(authorizedOrganization)) {
+                authorizedOrganization = OAuthConstants.AuthorizedOrganization.NONE;
+            }
+            insertTokenPrepStmt.setString(19, authorizedOrganization);
```

Comment:
After adding token-binding-ref, the authorized-organization should be added as shown in red color box 

<img width="883" alt="Screenshot 2023-10-15 at 17 49 54" src="https://github.com/wso2-extensions/identity-inbound-auth-oauth/assets/35717390/994e8efb-e8ec-426e-95f6-abb8cedaec40">